config/billing_rates.yml

```diff
@@ -63,6 +63,7 @@
 - { id: 27695566-1715-8978-9f84-b2cc5a91ee54, resource_type: IPAddress,                   resource_family: IPv4,                       location: latitude-fra,   unit_price: 0.0000992063, billed_by: duration, active_from: 2024-11-06T00:00:00Z }
 - { id: 15102862-1d57-4dd2-87ab-599f9b5b4732, resource_type: PostgresVCpu,                resource_family: standard-standard,          location: hetzner-fsn1,   unit_price: 0.0005518353, billed_by: duration, active_from: 2025-02-17T00:00:00Z }
 - { id: 08b86652-21b7-419a-b24e-e9149f98f6fa, resource_type: PostgresVCpu,                resource_family: standard-standard,          location: leaseweb-wdc02, unit_price: 0.0006696429, billed_by: duration, active_from: 2025-02-17T00:00:00Z }
+- { id: 289c6fce-02da-484d-a329-2829c7663f6a, resource_type: PostgresVCpu,                resource_family: standard-standard,          location: aws-us-west-2,  unit_price: 0.0009525360, billed_by: duration, active_from: 2025-02-17T00:00:00Z }
```

Comment:
We have `us-west-1` as location in the billing rates. Should we use similar pattern for both?